generators/server/templates/src/main/java/package/web/rest/MailResource.java.ejs

```diff
@@ -0,0 +1,42 @@
+<%#
+ Copyright 2013-2018 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.web.rest;
+
+import com.codahale.metrics.annotation.Timed;
+import <%=packageName%>.service.MailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/mail")
+public class MailResource {
```

Comment:
can we enchance the info endpoint to have this info rather than adding a new endpoint @PierreBesson should know about this

Reply:
Oh yes @deepu105 is right please remove this class if it is just to expose a property. You should rather write an InfoContributor in the jhipster-framework. Take inspiration from the ActiveProfileInfoContributor.